src/main/java/rx/internal/operators/OperatorUnsubscribeOn.java

```diff
@@ -59,6 +59,7 @@ public void onNext(T t) {
             @Override
             public void call() {
                 final Scheduler.Worker inner = scheduler.createWorker();
+                // FIXME should subscription returned be added to the subscriber composite
                 inner.schedule(new Action0() {
```

Comment:
This inner scheduler or the task can't be added because the unsubscription of child triggers the call and would prevent it from executing.